src/integration/pages/idxPage/crossPlatformTests.js

```diff
@@ -0,0 +1,75 @@
+import { runCommonCrossPlatformTests } from '../../common';
+
+export default () => {
+  runCommonCrossPlatformTests();
+
+  it('I can see the headline', () => {
+    const h1El = document.querySelector('h1');
+
+    expect(h1El).toBeInTheDocument();
+    expect(h1El.textContent).toBeTruthy();
+    expect(h1El.textContent).toMatchSnapshot();
+  });
+
+  it('I can see sections', () => {
+    const sections = document.querySelectorAll('section');
+
+    if (sections) {
+      sections.forEach(section => {
+        expect(section).toBeInTheDocument();
+
+        const h2El = section.querySelector('h2');
+
+        expect(h2El).toBeInTheDocument();
+        expect(h2El.textContent).toBeTruthy();
+        expect(h2El.textContent).toMatchSnapshot();
+      });
+    }
+  });
+
+  it('I can see Index Alsos', () => {
+    const topStories = document.querySelector(
+      '[aria-labelledby="Top-stories"]',
+    );
+
+    if (topStories) {
+      const indexAlsos = topStories.querySelector('[data-e2e=index-alsos]');
+
+      if (indexAlsos) {
+        const h4 = indexAlsos.querySelector('h4');
+        expect(h4.textContent).toMatchSnapshot();
+
+        if (window.SIMORGH_DATA) {
```

Comment:
Will this work on AMP (I wasn't sure whether amp pages have this or not)?

Reply:
No, I don't believe this will work on AMP. AMPs do not have the `SIMORGH_DATA`

Reply:
Ah yes, but won't fail because of if check - marking this as resolved